### PR TITLE
Move lineCounter app to azure-sdk repo

### PR DESCRIPTION
**Pull request by JoshLove-msft**
Changes from https://github.com/pakrym/LineCounter
- Use EventHubs Preview 6
- Use App Insights instead of Zipkin
- append .html to end of view file path to avoid server error involving accessing path with extensions like .cs

### CODE REVIEW

File: samples/linecounter/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (pakrym) [inline samples/linecounter/LICENSE]: rem

File: samples/linecounter/LineCounter.csproj [hunk 2]
```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <LangVersion>preview</LangVersion>
+    <UserSecretsId>67ca5d37-051b-4ab1-b013-c74a2d4edb76</UserSecretsId>
+    <RestoreSources>https://api.nuget.org/v3/index.json;https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-net/index.json</RestoreSources>
+    <ApplicationInsightsResourceId>/subscriptions/faa080af-c1d8-40ad-9cce-e1a450ca5b57/resourcegroups/jolov/providers/microsoft.insights/components/LineCounter</ApplicationInsightsResourceId>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="bin\**" />
```
Comment (pakrym) [inline samples/linecounter/LineCounter.csproj]: this shouldn't be required.

File: samples/linecounter/LineCounter.csproj [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.1.0" />
+    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.12.0" />
+    <PackageReference Include="Microsoft.Extensions.Azure" Version="1.0.0" />
+
+    <PackageReference Include="OpenTelemetry" Version="0.2.0-alpha.100" />
+    <PackageReference Include="OpenTelemetry.Exporter.ApplicationInsights" Version="0.2.0-alpha.100" />
+    <PackageReference Include="OpenTelemetry.Collector.Dependencies" Version="0.2.0-alpha.100" />
+    <PackageReference Include="OpenTelemetry.Collector.AspNetCore" Version="0.2.0-alpha.100" />
+  </ItemGroup>
+  <ItemGroup>
```
Comment (pakrym) [inline samples/linecounter/LineCounter.csproj]: rem

File: samples/linecounter/Program.cs [hunk 4]
```diff
@@ -0,0 +1,35 @@
+﻿using System;
```
Comment (pakrym) [inline samples/linecounter/Program.cs]: (c)

File: samples/linecounter/BlobProcessorHost.cs [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+        private readonly ILogger<LineCounterService> _logger;
+
+        private readonly BlobContainerClient _blobContainerClient;
+        private readonly EventHubProducerClient _resultsClient;
+
+        private readonly EventProcessorClient _processor;
+
+        public LineCounterService(
+            IAzureClientFactory<EventHubProducerClient> producerFactory,
+            BlobServiceClient blobServiceClient, ILogger<LineCounterService> logger,
```
Comment (pakrym) [inline samples/linecounter/BlobProcessorHost.cs]: format

**Comment by JoshLove-msft** [comment]
FYI @jongio @kaerm

File: samples/linecounter/BlobProcessorHost.cs [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+            _processor.ProcessEventAsync += ProcessEvent;
+            _processor.ProcessErrorAsync += (eventArgs) =>
+            {
+                _logger.LogError(eventArgs.Exception, "Error in " + eventArgs.PartitionId);
+                return Task.CompletedTask;
+            };
+            await _processor.StartProcessingAsync();
+        }
+
+        //private async Task ProcessEvents(PartitionContext arg1, IEnumerable<EventData> events)
```
Comment (pakrym) [inline samples/linecounter/BlobProcessorHost.cs]: rem

File: samples/linecounter/Connected Services/Application Insights/ConnectedService.json [hunk 7]
```diff
@@ -0,0 +1,4 @@
+﻿{
```
Comment (pakrym) [inline samples/linecounter/Connected Services/Application Insights/ConnectedService.json]: Remove this file

File: samples/linecounter/Controllers/HomeController.cs [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+        [HttpGet("status/{name}.html")]
+        public async Task<string> Status(string name)
+        {
+            var properties = await _blobContainerClient.GetBlobClient(name).GetPropertiesAsync();
+            properties.Value.Metadata.TryGetValue("whitespacecount", out var count);
+
+            return count ?? "-1";
+        }
+    }
+
```
Comment (pakrym) [inline samples/linecounter/Controllers/HomeController.cs]: rem

**Review by pakrym — APPROVED** [review]

**Pull request merged**